testsuite/features/step_definitions/command_steps.rb

```diff
@@ -42,24 +42,6 @@
   step 'I install a salt pillar top file for "salt_bundle_config, disable_local_repos_off" with target "*" on the server'
 end
 
-Then(/^"([^"]*)" should communicate with the server using public interface/) do |host|
-  node = get_target(host)
-  _result, return_code = node.run("ping -c 1 -I #{node.public_interface} #{get_target('server').public_ip}", check_errors: false)
-  unless return_code.zero?
-    sleep 2
-    puts 're-try ping'
-    node.run("ping -c 1 -I #{node.public_interface} #{get_target('server').public_ip}")
-  end
-  get_target('server').run("ping -c 1 #{node.public_ip}")
-end
-
-Then(/^"([^"]*)" should not communicate with the server using private interface/) do |host|
-  node = get_target(host)
-  node.run_until_fail("ping -c 1 -I #{node.private_interface} #{get_target('server').public_ip}")
-  # commented out as a machine with the same IP address might exist somewhere in our engineering network
-  # get_target('server').run_until_fail("ping -c 1 #{node.private_ip}")
-end
-
```

Comment:
I miss some extended information in the PR description about the strategy we followed here, and why this is not anymore necessary.

Reply:
It is not called anymore in non-containerized proxy context.

Since we don't have traditional client and salt minion on the private network, we have nothing to ping anymore... As a matter of fact, in a non-containerized setup, the private network will be completely empty at the beginning.

For the containerized proxy, I will re-add ping checks, but with a slightly different name, and with a completely different implementation, as we will now ping the DHCP and DNS branch server, which will be the only VM on the private network at the beginning.

The non-containerized setup with remain without ping sanity checks, it will go away soon enough in Uyuni, and for 4.3 it never caught an error in the past, so...

Reply:
It would be awesome if you can explain this in the PR description.

Reply:
ok